docs/admin/guides/build-image.md

```diff
@@ -6,47 +6,79 @@
     guaranteed.
 
 Users can add new images to a container repository by uploading a Containerfile. The syntax for
-Containerfile is the same as for a Dockerfile. The same REST API endpoint also accepts a JSON
-string that maps artifacts in Pulp to a filename. Any artifacts passed in are available inside the
-build container at `/pulp_working_directory`.
+Containerfile is the same as for a Dockerfile.
 
-## Create a Repository
+To make a file available during the build process, we need to add it to a versioned file repository,
+then pass it as the `build_context` query parameter of the build container API, and finally specify the
+```
+ADD/COPY <file relative-path> <location in container>
+```
+instruction in the Containerfile.
```

Comment:
1. What is a file? We should re-phrase it to something like "To pass arbitrary files or artifacts to the image building context, the `build_context` property can be provided in the request payload, containing a reference to a file repository."
2. `build_context` is not a [query parameter](https://docs.oracle.com/en/cloud/saas/cx-commerce/21b/ccdev/rest-api-query-parameters.html). It is a "JSON property" in the request body, see [the difference](https://stackoverflow.com/a/42462714).